.buildkite/scripts/build_kibana.sh

```diff
@@ -8,7 +8,7 @@ echo "--- Build Kibana Distribution"
 if [[ "${GITHUB_PR_LABELS:-}" == *"ci:build-all-platforms"* ]]; then
   node scripts/build --all-platforms --skip-os-packages
 elif [[ "${GITHUB_PR_LABELS:-}" == *"ci:build-os-packages"* ]]; then
-  node scripts/build --all-platforms
```

Comment:
Why not just make cross-compiling the default?

Reply:
It requires the qemu toolchain to work https://github.com/elastic/kibana/pull/128272